### DISCUSSION TITLE
Suggest log websites to issue tracker form

### DISCUSSION THREAD

**Discussion by g0urra** [NillerMedDild/Enigmatica6#3828] [Ideas]
When adding an issue to the issue tracker with the form, there is no website to suggest a site. Most users seem to add link to pastebin, which limits how long the text can be.

I would suggest adding either https://gist.github.com/ or https://paste.gg/ to the form at this part.

![bild](https://user-images.githubusercontent.com/12349921/144333753-9226891c-659a-4ecb-9fd6-09676681b553.png)

**Comment by NielsPilgaard** [comment]
Good idea, added :)
